### PR TITLE
gc_manager: add a log to output gcSafePoint update record.

### PR DESCRIPTION
**Pull request by asddongmen**
### What is changed and how it works?

1. Add a log to output gcSafePoint update record.
    The purpose is to better locate and fix some issues related to gcSafePoint. 
    #2191 
    Such type issues is very hard to repreduce and only occur in pressure test，output log here helps to better locate the problem the next time it reappears.
2. Refine some comments.

### Check List 

Tests 

 - No code

Code changes

 - None

Side effects

 - None

Related changes

 - Need to cherry-pick to the release branch

### Release note

- No release note

### CODE REVIEW

**Review by ben1009 — APPROVED** [review]

File: cdc/owner/gc_manager.go [hunk 1]
```diff
@@ -94,6 +94,9 @@ func (m *gcManager) updateGCSafePoint(ctx cdcContext.Context, state *model.Globa
 	failpoint.Inject("InjectActualGCSafePoint", func(val failpoint.Value) {
 		actual = uint64(val.(int))
 	})
+	if actual == minCheckpointTs {
+		log.Info("update gc safe point success", zap.Uint64("gcSafePointTs", minCheckpointTs))
```
Comment (overvenus) [inline cdc/owner/gc_manager.go:98]: What's the frequency of the log?

**Review by overvenus — COMMENTED** [review]
Rest LGTM

File: cdc/owner/gc_manager.go (hunk 1 above)
Comment (amyangfei) [reply cdc/owner/gc_manager.go:98]: At least one minute. In fact PD log contains the service safepoint update information

**Review by overvenus — APPROVED** [review]

**Comment by asddongmen** [comment]
> /merge

need more approved

**Review by amyangfei — APPROVED** [review]

**Pull request merged**